r/tests/testthat/test-Table.R

```diff
@@ -359,9 +359,9 @@ test_that("table() auto splices (ARROW-5718)", {
 })
 
 test_that("Validation when creating table with schema (ARROW-10953)", {
-  tab <- Table$create(data.frame(), schema = schema(a = int32()))
-  skip("This segfaults")
-  expect_identical(dim(as.data.frame(tab)), c(0L, 1L))
+  expect_error(Table$create(data.frame(), schema = schema(a = int32())))
```

Comment:
Should we assert the error messages, just to make sure it looks like we want?

Reply:
Done in 50badd4e1af999cca6ff9f173d2d6a797d5407b9